kernelci.org/content/en/docs/org/tsc/votes.md

```diff
@@ -9,14 +9,49 @@ outcome and a more detailed section listing each individual vote.
 
 ## Summary
 
-| Date                      | Motion                               | Status   |
-|---------------------------|--------------------------------------|----------|
-| [2022-06-09](#2022-06-09) | Add TSC member: Denys                | Approved |
-| [2022-02-10](#2022-02-10) | Remvove TSC member: Lakshmipathi     | Approved |
-| [2021-07-13](#2021-07-13) | Add TSC member: Alice                | Approved |
+| Date                      | Motion                                                          | Status   |
+|---------------------------|-----------------------------------------------------------------|----------|
+| [2021-12-08](#2022-12-08) | Adopt the currently documented TSC rules as a starting point    | Approved |
+| [2021-12-08](#2022-12-08) | Add mgalka to the GitHub admins team                            | Approved |
+| [2021-12-08](#2022-12-08) | List of SysAdmin resources                                      | Approved |
+| [2022-06-09](#2022-06-09) | Add TSC member: Denys                                           | Approved |
+| [2022-02-10](#2022-02-10) | Remvove TSC member: Lakshmipathi                                | Approved |
+| [2021-07-13](#2021-07-13) | Add TSC member: Alice                                           | Approved |
 
 ## Votes
 
+### 2022-12-08
+
+**Motion: Confirm a list of the components to which each member of the sysadmin subgroup should have access**
+* Result: approved
+* Effective date: 2022-12-08
+* Voting method: regular TSC meeting/Mail
+* Voting members: mgailka, denis, gtucker, spbnick, broonie, alicef, Khilman
+* Vote count: 7/9 voting, 7/7 in favour
+* Links: 
+  * [(Docs)2022-12-08 TSC Monthly](https://docs.google.com/document/d/1aoNpi69WRhn1nENbdAY1CPXYVkufbVnG_lFoGfA0aM4/edit#heading=h.g3k9k5jf9amw)
```

Comment:
Are you sure this link is correct and actually useful?  It seems to be opening the document at the wrong page and I need to scroll down to find the TSC notes.

Moving forward, I think there are things in the TSC meeting minutes that shouldn't really be shared publicly.  For example, if we're discussing various replies in response to a RFP then it might be a bit messy if people learnt about the decisions this way rather than via the board.  I would prefer to drop this link tbh and just have a link to the email archive.

Reply:
just wait loading and it will take you to 2022-12-08 TSC Monthly meetings

Reply:
anyway updated with gdocs link